### DISCUSSION TITLE
is it possible to use the k9s to manage the kubesphere

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [kubesphere/kubesphere#6147] [Q&A] [answered]
I have already installed the k9s in the remote server:

```
bash-5.1# k9s version
 ____  __.________
|    |/ _/   __   \______
|      < \____    /  ___/
|    |  \   /    /\___ \
|____|__ \ /____//____  >
        \/            \/

Version:    0.24.2
Commit:     a952806e

```

when I login the k9s the dashboard just have nothing。 Is it possible to use k9s  to manage the cluster? i am in hte kubectl environment in the web console.

**Comment by calvinyv** [comment, marked as answer]
currently, KubeSphere doesn't support k9s.
